### DISCUSSION TITLE
How do I overwrite the cursor color?

### DISCUSSION THREAD

**Discussion by mabq** [rose-pine/neovim#115] [Q&A] [answered]
How do you overwrite the cursor color?

```lua
require('rose-pine').setup({
	highlight_groups = {
		CursorLine = { bg = 'surface' }, -- this works
		Cursor = { fg='text', bg='gold' }, -- this does not work
	}
})
```

For some reason `CursorLine` works but `Cursor` does not.

Thank you for an amazing theme!

**Comment by mvllow** [comment, marked as answer]
Unfortunately (to my knowledge) `Cursor` only applies to GUI applications, but not the terminal. I use kitty terminal and Neovim uses the cursor colour set in my terminal theme.

```ini
# Example kitty.conf
cursor                   #ea9d34 # Rosé Pine Dawn gold
cursor_text_color        #e0def4 # Rosé Pine text
```

**Reply by mabq** [reply]
Thank you so much!
